lib/domain.js

```diff
@@ -46,8 +46,11 @@ const { WeakReference } = internalBinding('util');
 
 // Overwrite process.domain with a getter/setter that will allow for more
 // effective optimizations
+// Never directly access _domain, use the getter to ensure a consistent hook
```

Comment:
Totally optional and unnecessary nit. 100% non-blocking. But you know, a comma-splice is a comma-splice.

```suggestion
// Never directly access _domain. Use the getter to ensure a consistent hook
```